truffle/test/05_eth_api.js

```diff
@@ -2,9 +2,27 @@
  * - Execute 100 transaction in parallel
  */
 
+const Web3 = require('web3');
+const { Debug } = require('web3-eth-debug');
+
+const { extractEnvAccountAndPwd } = require('./utils');
+const HelloBlockchainWorld = artifacts.require("HelloBlockchainWorld");
+
 describe('Ethereum API', () => {
+  let _web3;
+  let _debug;
+  let ROOT_ACCOUNT = extractEnvAccountAndPwd(process.env.NETWORK).from;
+
+  it("should initialize library web3", async () => {
+    _web3 = new Web3(web3._provider.host, null, {
+      defaultAccount: ROOT_ACCOUNT,
+      defaultGasPrice: "500000000000"
```

Comment:
can we add this to the .env file? And read it from all the tests or usages? This value seems to be repeated quiet a lot and we may change it soon. Will be easier to refactor.

Reply:
It is not the `.env`, it is the truffle.config and I tried and it is not possible unless I add logic on the migrations to do that and it seems annoying to do so

Reply:
if it's in the truffle config then u don't have to define it here btw. but is a little thing, will be easy to change one way or the other. approved